benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -127,7 +127,7 @@
   {"LiveFrom": "10/09/2019", "CaseTypeID": "Benefit", "CaseEventID": "attachScannedDocs", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRU"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "caseUpdated", "UserRole": "caseworker-sscs", "CRUD": "R"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "caseUpdated", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
-  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "caseUpdated", "UserRole": "caseworker-sscs-judge", "CRUD": "R"},
+  {"LiveFrom": "20/09/2022", "CaseTypeID": "Benefit", "CaseEventID": "caseUpdated", "UserRole": "caseworker-sscs-registrar", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "caseUpdated", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
```

Comment:
I think sscs-systemupdate shouldn't have this permission

Reply:
That should be fine, the system user does make case updates